build/dependencies.yaml

```diff
@@ -18,7 +18,7 @@ dependencies:
 
   # agnhost: bump this one first
   - name: "agnhost"
-    version: "2.32"
+    version: "2.33"
```

Comment:
Below these lines it says "agnhost: dependents" and it still has v2.32, do we need to update that too?

Reply:
We update that later. First, we need to merge this PR, so the **postsubmit** jobs will build agnhost:2.33, then we promote the image from staging, then we send a PR that updates test/utils/image/manifest.go.

Reply:
Got it, thanks